.yarnrc.yml

```diff
@@ -0,0 +1,5 @@
+yarnPath: .yarn/releases/yarn-4.7.0.cjs
```

Comment:
Can't we just get yarn via corepack?

Reply:
Corepack is an experimental feature and requires additional opt-in, and the Node.js core team is moving towards ending support for Corepack.

Currently `yarnPath` is the simplest and most reliable way to achieve [hermeticity](https://bazel.build/basics/hermeticity).